main.py

```diff
@@ -10,9 +10,19 @@ def check_id_and_pin(user_id, user_pin):
 
 
 def get_number_from_input(text):
-    num = input(text)
+    try:
+        num = input(text)
+        return int(num)
+    except ValueError:
+        print("Use 4 digits!")
 
-    return int(num)
+
+def validation_credential(user_id, user_pin):
+    if len(str(user_id)) == 4 and len(str(user_pin)) == 4:
+        return True
```

Comment:
I'm cheking only the length of args. I think it looks better, but it's count as the wrong attemp now. 
Just because we always have int. Even if you input the str, it converts to the int